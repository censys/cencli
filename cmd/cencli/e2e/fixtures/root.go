package fixtures

import (
	"testing"
	"time"

	"github.com/censys/cencli/cmd/cencli/e2e/fixtures/golden"
)

var RootFixtures = []Fixture{
	{
		Name:      "base",
		Args:      []string{},
		ExitCode:  0,
		Timeout:   5 * time.Second, // give more time since this will be run on all platforms
		NeedsAuth: false,
		Assert: func(t *testing.T, stdout, stderr []byte) {
			assertGoldenFile(t, golden.RootStdout, stdout, 0)
		},
	},
}
