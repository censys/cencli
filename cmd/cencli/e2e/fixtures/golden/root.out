censys - The Censys CLI

The Censys CLI brings the authority of internet intelligence to your terminal. 
Leverage the Censys Platform to analyze assets, perform searches, hunt threats,
all from the command line.                                                     

Get started by exploring the available commands below.

Available Commands:
  aggregate   Aggregate results for a Platform search query
  censeye     Analyze a host and generate pivotable queries with rarity bounds
  completion  Generate shell completion scripts
  config      Manage configuration
  history     Retrieve historical data for hosts, web properties, and certificates
  search      Execute a search query across Censys data
  version     Print version information
  view        Retrieve information about hosts, certificates, and web properties

Run "censys [command] --help" for help with a specific command.

For more information, please refer to the documentation: https://docs.censys.com/docs/platform-cli

